apps/hip/utils.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import Case, CharField, F, Value, When
+
+from wagtail.core.models import Page
+from wagtail.documents.models import Document
+
+
+def get_most_recent_objects(object_count=10):
+    """
+    Return the most recently updated objects from a number of models.
+
+    Currently, we return the most recently updated models from:
+     - Pages
+     - Documents
+
+    In order to make it easier to get data from each returned object (in a template),
+    several fields are annotated on each object:
+     - name
+     - updated_at
+     - type_of_object
+     - model_name
+    Note: objects are returned in a list. It is also possible to combine querysets
+    together using queryset.union(), but it is difficult to do so for different
+    models without using .values(), which means that each object in the queryset
+    loses its methods. In order to preserve the methods, objects are combined in
+    a list.
+    """
+    # Get the most recent Page objects.
+    pages = Page.objects.exclude(latest_revision_created_at__isnull=True).annotate(
```

Comment:
I think we'll want `Page.objects.live()...` because it is currently retrieving draft pages too.

Reply:
✔️